### PR TITLE
Catalyst and helper guides

### PR DESCRIPTION
**Pull request by meldra**
adds catalyst.md for opers and ops
adds helpers.md for regular helpful folk

### CODE REVIEW

File: content/_guides/catalyst.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+Removing people should ideally not be a permanent thing. People make mistakes
+and have bad days. One mistake or bad day shouldn't be the end of their
+participation in your channel.
+
+Also, permanently excluding users from a channel can lead to them seeking a
+way of evading into the channel, becoming even more problematic.
+
+It's an excellent idea for your channel to have a way for people to appeal a
+ban and negotiate being allowed back. Consider having an "ops" channel; most
+channels use #<channelname>-ops for this. Discourage non-ops from lingering
```
Comment (glguy) [inline content/_guides/catalyst.md:152]:
```suggestion
channels use #\<channelname\>-ops for this. Discourage non-ops from lingering
```

File: content/_guides/helpers.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+If _you_ need the factoid to remind you of the information, and the bot does
+not give factoids in PM, explain as you're doing it. This also teaches them
+how to use the bot. Also, don't just leave them hanging with the answer! Check
+it yourself to ensure it has the information you think it does, and prompt the
+user to confirm if it covers what they want.
+
+## When to stop engaging
+
+When a situation requires intervention and your own attempts to diffuse it to
```
Comment (emersonveenstra) [inline content/_guides/helpers.md:148]:
```suggestion
When a situation requires intervention and your own attempts to diffuse it do
```

File: content/_guides/catalyst.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+If you have ever had a technical problem, you know how overwhelming it can be.
+Sometimes people seeking help will get very frustrated and might have a temper
+or be rude. Patient and efficient help can be vital in avoiding conflict in
+these situations.
+
+This guide will outline some good practices for both operators and helpers for
+interacting with people. It will also discuss some common situations and ideal
+ways of responding.
+
+## Only be "opped up" when necessary
```
Comment (Mikaela) [inline content/_guides/catalyst.md:18]: I have critique on this part. This doesn't address services occassionally being down during which new ops cannot be reveived and doesn't communicate to people who don't already know how to get op attention.

https://mikaela.info/blog/english/2015/04/01/keep-the-ops-opped.html

Comment (Mikaela) [reply content/_guides/catalyst.md:18]: Additionally Matrix users have additional difficulties seeing the op status of users as the current instance of the bridge only syncs ops to the other side and they need to use +O for syncing state as much as they can.

File: content/_guides/catalyst.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+Most channels have a select few operators who are responsible for managing the
+culture of the channel. These are the people with `+o` flags. Most clients
+display a `@` next to their nick. Libera.Chat does not have "halfops" flags
+like some other networks.
+
+We recommend that operators in channels refrain from being "opped up" unless
+it is necessary for immediate channel management activities like changing a
+locked topic or removing a problematic user. We discourage using the `+O`
```
Comment (Mikaela) [inline content/_guides/catalyst.md:27]: I find the word "discourage" here as triggering negative reaction as I consider my critique on avoiding being opped as valid and I find it coming across as "no matter your reasoning to not follow this part, you must be a horrible person if you didn't follow this order".

File: content/_guides/catalyst.md (hunk 3 above)
Comment (jesopo) [reply content/_guides/catalyst.md:18]: we try to warn users ahead of services downtime and people can (usually) check chanserv flags to find ops. the social benefit of not wearing far outweighs these points

File: content/_guides/catalyst.md (hunk 4 above)
Comment (jesopo) [reply content/_guides/catalyst.md:27]: you're projecting a lot of meaning on to "discourage"

File: content/_guides/catalyst.md (hunk 3 above)
Comment (jesopo) [reply content/_guides/catalyst.md:18]: (additionally, I don't want to make compromises on productively handling users for the sake of a matrix shortcoming)

Comment (Mikaela) [reply content/_guides/catalyst.md:18]: Clients have a lot of differences in how they show globals or wallops, so they can be missed and it's possibly for ops to just not be on between the time of scheduled downtime announcement and beginning.

File: content/_guides/catalyst.md (hunk 4 above)
Comment (Mikaela) [reply content/_guides/catalyst.md:27]: I am aware. My request would be to rephrase this in a way that doesn't include a negation. E.g.

* remove this last phrase entirely.
* we recommend giving ops the AOP template.
* this can be achieved by using the AOP template.

File: content/_guides/catalyst.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+and have bad days. One mistake or bad day shouldn't be the end of their
+participation in your channel.
+
+Also, permanently excluding users from a channel can lead to them seeking a
+way of evading into the channel, becoming even more problematic.
+
+It's an excellent idea for your channel to have a way for people to appeal a
+ban and negotiate being allowed back. Consider having an "ops" channel; most
+channels use `#<channelname>-ops` for this. Discourage non-ops from lingering
+in the channel, so people who approach you can do so with some privacy. When
```
Comment (Mikaela) [inline content/_guides/catalyst.md:153]:
```suggestion
channels use `#<channelname>-ops` for this. Encourage non-ops to leave the channel once
their case has been handled, so people who approach you can do so with some privacy. When
```

I think this would be a more positive way to communicate the same thing.

**Review by Mikaela — COMMENTED** [review]
Continuing the discussion in catalysts. In general I think it's more positive than the helpers guide which at times seems to bluntly begin with negative phrases on what to not do instead of what to do.

File: content/_guides/catalyst.md (hunk 3 above)
Comment (jesopo) [reply content/_guides/catalyst.md:18]: downtime is typically short, find network staff if you missed the wallops and have an urgent problem. it seems like a one in a million problem and certainly shouldn't discard a tried and true aspect of nonescalation

Comment (Mikaela) [reply content/_guides/catalyst.md:18]: Sorry, I forgot in my previous comment, could there be a link somewhere on this section to another page explaining how to get connection to an operator whose flags aren't visible (including `/msg chanserv flags #channel` and how to connect e.g. account Mikaela to Aminda using information available through ChanServ or `/who`?

Comment (jesopo) [reply content/_guides/catalyst.md:18]: channels without viewable access lists typically don't want their ops found for whatever reason, I think it would be irresponsible to advise people on how to get around that desire

Comment (Mikaela) [reply content/_guides/catalyst.md:18]: I am asking about channels with public access list. 

Using a real life example of a channel which has a compromise solution by only having a bot opped:

* There is spam on ##jrpg (and in this case I am pretending it's a much larger channel than it actually is)
* New IRC user doesn't know who are ops and how to reach them
* More advanced IRC user knows to /msg chanserv flags ##jrpg list
* The access list contains account name `Mikaela`. However currently that nickname doesn't exist on the channel. How will the user know that `Mikaela` is actually using name `Aminda`?

If there is no page explaining this issue, should I open a new issue to request one to be created which could then be linked from here?

Comment (Fuchs) [reply content/_guides/catalyst.md:18]: We won't be able to handle every single potential edge case, and it is recommendations, not fixed rule. We think that the benefits of not being opped all the time are bigger than the issues and caveats, channel operators disagreeing are free to wear their op hat.  On finding channel ops, it has advantages and disadvantages, e.g. if there are three people wearing an op hat but all of them are afk, that is not helpful for people being new to IRC either, it is actually potentially frustrating. 

tl;dr: we know there are downsides to each variant, but we prefer this one, thus we put it in our recommendation and I think the wording is fine as is.

Comment (jesopo) [reply content/_guides/catalyst.md:18]: the point about instructions on querying `CS FLAGS` is a reasonable one though

Comment (Fuchs) [reply content/_guides/catalyst.md:18]: I think it creates more edge cases (private versus public) and additional documentation needs (which flag is what, how do I reach someone who is not online, how do I find someone by account instead of nick) that will make it a lot harder to read, so if, then it should imho be split into separate documentation and linked.

Comment (jesopo) [reply content/_guides/catalyst.md:18]: yes, Mikaela suggested a separate document

Comment (Mikaela) [reply content/_guides/catalyst.md:18]: @emersonveenstra (in https://github.com/Libera-Chat/libera-chat.github.io/issues/60#issuecomment-901961470) points out that it's documented in https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted so could that be linked somewhere here?

Comment (meldra) [reply content/_guides/catalyst.md:18]: As Fuchs points out above, we still need additional documentation which we'll then link to from various places. One specific page we need is a brief walk-through on advising what flags are actually needed for non-founder ops when setting up a new channel. One of the last pages i did, hurriedly with the intention of rewriting it before launch, was https://libera.chat/guides/creatingchannels which is a bit dire really and could do with explaining how to set up ops in it too. How to check your ops would be part of the checklist for that and probably an ideal place to link to from this.

Until that's done, we have already mentioned how to view flags on https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted but it's not an ideal place to throw people from this document in particular.

File: content/_guides/catalyst.md (hunk 5 above)
Comment (meldra) [reply content/_guides/catalyst.md:153]: Well, now it's a bit specific to those who even had a case to start with and not also tourists. I'll come up with a compromise.

Comment (meldra) [reply content/_guides/catalyst.md]: Does the new wording i've committed address your concern?

File: content/_guides/catalyst.md (hunk 4 above)
Comment (meldra) [reply content/_guides/catalyst.md:27]: Discouraging isn't the same as forbidding. We don't forbid its use. You're not breaking any rules by doing it. You'd just be using (by our philosophy) non-best practices. Sometimes that might cause discomfort at doing something less than 100% correct, but that's not on us.

File: content/_guides/catalyst.md (hunk 3 above)
Comment (meldra) [reply content/_guides/catalyst.md:18]: Actually on second thoughts, a faq section of its own might be better, and have that faq section pointed to from the getting unbanned instructions. since pointing people seeking ops for a problem situation might not find it encouraging to be punted to instructions on how a banned person could undermine their efforts.

File: content/_guides/catalyst.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+these situations.
+
+This guide will outline some good practices for both operators and helpers for
+interacting with people. It will also discuss some common situations and ideal
+ways of responding.
+
+## Only be "opped up" when necessary
+
+Most channels have a select few operators who are responsible for managing the
+culture of the channel. These are the people with `+o` flags. Most clients
```
Comment (swantzter) [inline content/_guides/catalyst.md:21]: I wonder if we should make a clearer distinction between ChanServ flags and IRC modes, since there are often confusion about it. Maybe as simple as spelling out "people with the ChanServ flag +o" and later being voice (having mode +v) or similar

File: content/_guides/catalyst.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+co-worker or a classmate. Users may view your responses as orders, threats, or
+trick questions when you do not intend them to be.
+
+Channel Ops should avoid demonstrating hierarchy and interact as regular
+participants in their channels as much as possible. Try not to be someone who
+only appears when things are heated. Having a rapport with your users will
+make it easier to de-escalate heated situations, making it easier to avoid
+having to forcefully remove people from your community.
+
+### Why is #libera different?
```
Comment (swantzter) [inline content/_guides/catalyst.md:47]:
```suggestion
### Why is `#libera` different?
```

Styleguide

File: content/_guides/catalyst.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+
+You may have noticed that the #libera channel doesn't follow this guideline.
```
Comment (swantzter) [inline content/_guides/catalyst.md:49]:
```suggestion
You may have noticed that the `#libera` channel doesn't follow this guideline.
```

Styleguide

File: content/_guides/catalyst.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+
+The #libera channel is relatively large and is a focal point for botnets and
```
Comment (swantzter) [inline content/_guides/catalyst.md:51]:
```suggestion
The `#libera` channel is relatively large and is a focal point for botnets and
```

Styleguide

File: content/_guides/catalyst.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+other nefarious behavior. It uses channel mode +z when dealing with unwanted
```
Comment (swantzter) [inline content/_guides/catalyst.md:52]:
```suggestion
other nefarious behavior. It uses channel mode `+z` when dealing with unwanted
```

Styleguide

File: content/_guides/catalyst.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+messages, which requires staff to be opped, so they don't miss messages.
+
+Additionally, people often come to #libera to find network staff with
```
Comment (swantzter) [inline content/_guides/catalyst.md:55]:
```suggestion
Additionally, people often come to `#libera` to find network staff with
```

Styleguide

File: content/_guides/catalyst.md [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+participation in your channel.
+
+Also, permanently excluding users from a channel can lead to them seeking a
+way of evading into the channel, becoming even more problematic.
+
+It's an excellent idea for your channel to have a way for people to appeal a
+ban and negotiate being allowed back. Consider having an "ops" channel; most
+channels use `#<channelname>-ops` for this. Request that non-ops leave the
+channel if they do not have an immediate issue, so that people can raise
+their issues with some privacy. When talking to banned users in an -ops
```
Comment (swantzter) [inline content/_guides/catalyst.md:156]:
```suggestion
their issues with some privacy. When talking to banned users in an `-ops`
```

I feel like this should be treated the same as a channel name since it's a partial channel name

File: content/_guides/catalyst.md [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+channel, de-escalation tools and building rapport are critically important.
+
+If many channel operators speak to a person at once in an -ops channel, it can
```
Comment (swantzter) [inline content/_guides/catalyst.md:159]:
```suggestion
If many channel operators speak to a person at once in an `-ops` channel, it can
```

If nothing else just to make it more visually clear that the - isn't used for sentence pausing

File: content/_guides/catalyst.md [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+seem like "ganging up on" the person. You may want to informally limit how
+many ops are involved in such discussions at once.
+
+Operator channels also allow people a place to report issues without
+escalating situations in the channel. It is a good idea to advertise it in the
+channel's topic. If people cannot get into your ops channel, they cannot use
+it, so banning from that should be very rare.
+
+It is also a good idea to ensure that people can use ChanServ to find nicks of
```
Comment (swantzter) [inline content/_guides/catalyst.md:168]:
```suggestion
It is also a good idea to ensure that people can use `ChanServ` to find nicks of
```

Styleguide

File: content/_guides/helpers.md [hunk 15]
```diff
@@ -0,0 +1,165 @@
+---
+title: Advice for Helpers
+category: chanop
+---
+
+Communities are healthiest when everyone works together to maintain the space
+and shape the culture. If we leave all the emotional labor for Operators to
```
Comment (swantzter) [inline content/_guides/helpers.md:7]: Should "Operators" be capitalised here? It feels odd to me but i don't know English

File: content/_guides/helpers.md [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+deal with, they will be more likely to
+[burn out](catalyst#remember-to-take-breaks-to-avoid-burn-out).
+
+These are some things non-Operator helpers can keep in mind during challenging
```
Comment (swantzter) [inline content/_guides/helpers.md:11]: -//-

File: content/_guides/helpers.md [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+situations to avoid making more work for others.
+
+## Let small things slide
+
+The best response to minor annoying behavior is often to ignore it. Drawing
```
Comment (swantzter) [inline content/_guides/helpers.md:16]: One method we use a lot in sports, that I wonder is worth mentioning, or if it's too complex to get across well, is to not comment on mistakes the first two-three times since the person is likely to self-reflect on what they did, and on the third/fourth time you mention it in some relative privacy and/or in a very non-confronting/non-accusing way, and ask if they want more detail and or help.

Comment (swantzter) [reply content/_guides/helpers.md:16]: Essentially letting small things slide forever might not be the best idea always, if it results in a slow but steady escalation/stretching the limits

File: content/_guides/helpers.md [hunk 18]
```diff
@@ -0,0 +1,165 @@
...
+
+In a support channel, helping a user with their problem is much more important
+than trying to make them ask in the "right" way. So unless it becomes a habit
+or makes the channel uncomfortable, it's not worth derailing the support
+process.
+
+### Typos and non-fluent language
+
+Don't draw attention to someone else's typos or pick on them for their use of
+abbreviations like 'u', unless they specifically request help with wording.
```
Comment (swantzter) [inline content/_guides/helpers.md:39]: Should this be single-quotes, feels like the rest of the doc uses double quotes as the primary quote. Admittedly not entirely sure on English quote rules so might turn out to be the ultimate irony to comment this below this header

File: content/_guides/helpers.md [hunk 19]
```diff
@@ -0,0 +1,165 @@
...
+to `/whois` them to find out if they are identified or not:
+
+```irc
+/whois Nickname
+```
+
+If they are identified, there will be a line saying so. It looks like this:
+
+```irc
+[Nickname] is logged in as Accountname
```
Comment (swantzter) [inline content/_guides/helpers.md:90]:
```suggestion
<nickname> is logged in as Accountname
```

I'd use `<>` to signify that it's "required"/always present, maybe an overfluffy interpretation of the styleguide

File: content/_guides/helpers.md [hunk 20]
```diff
@@ -0,0 +1,165 @@
...
+11:49 -- Channel #windows created on Wed, 19 May 2021 07:36:22
+11:49 -- [#windows] $~a quieted by copper.libera.chat on Mon, 14 Jun 2021 10:49:19
+11:49 -- [#windows] End of Channel Quiet List
+```
+
+If the user needs to be identified, and they are not, they might need
+reminding. You can do this when redirecting them politely.
+
+```irc
+<You> I don't know much about Windows, but the people in #windows will! You will need to be registered to talk in there. If you have an account, see '/msg nickserv help identify'. If not, see '/msg nickserv help register' :)
```
Comment (swantzter) [inline content/_guides/helpers.md:117]: I wonder if we should be consistent with how atheme docs present commands here, and use same capitalisation in the example. Also would it be nicer to use a channel name for a FOSS/PDP project rather than windows?


```suggestion
<You> I don't know much about Windows, but the people in #windows will! You will need to be registered to talk in there. If you have an account, see '/msg NickServ HELP IDENTIFY'. If not, see '/msg NickServ HELP REGISTER' :)
```

File: content/_guides/helpers.md [hunk 21]
```diff
@@ -0,0 +1,165 @@
...
+
+## When to stop engaging
+
+When a situation requires intervention and your own attempts to diffuse it do
+not seem to be working, it is sometimes best to stand down and stop
+contributing to the noise. Instead, try to [seek out a channel operator](faq#how-to-find-out-who-runs-a-channel)
+quietly, such as with private messages or in an `-ops` channel if there is
+one.
+
+If you know that someone else around is an Operator, and they seem to be
```
Comment (swantzter) [inline content/_guides/helpers.md:154]: Same as above with capitalisation of "operator"

File: content/_guides/helpers.md [hunk 22]
```diff
@@ -0,0 +1,165 @@
...
+paying attention to the channel while the disruption is occurring, they might
+already be talking to the person in private. Check with the Operator if you're
```
Comment (swantzter) [inline content/_guides/helpers.md:156]: -//-

File: content/_guides/helpers.md [hunk 23]
```diff
@@ -0,0 +1,165 @@
...
+unsure before also trying to intervene.
+
+When an Operator starts handling the situation, continuing to engage with the
+person who is misbehaving may derail the Operator's efforts, as the person may
```
Comment (swantzter) [inline content/_guides/helpers.md:160]: -//- *2

File: content/_guides/helpers.md [hunk 24]
```diff
@@ -0,0 +1,165 @@
...
+become defensive at being "ganged up on".
+
+If necessary, use the `/ignore` functionality in your client to stop
+contributing. You don't want to cause the situation to escalate, or you might
+also end up in trouble.
```
Comment (swantzter) [inline content/_guides/helpers.md:165]: Something about this wording feels hostile to me "don't escalate or there will be CONSEQUENCES" *thunder roars in the background*

Not sure if I have a better suggestion, maybe something along the lines of "there may be instances where stepping in might lead to the situation escalating and you becoming a part of the issue". But not sure that sounds any better

File: content/_guides/catalyst.md (hunk 7 above)
Comment (meldra) [reply content/_guides/catalyst.md:47]: oh wasn't aware ticks were legal in headers or i'd have done that

File: content/_guides/helpers.md (hunk 17 above)
Comment (meldra) [reply content/_guides/helpers.md:16]: yeah. i think elsewhere or previously here it says "initially ignore it"

File: content/_guides/helpers.md (hunk 19 above)
Comment (meldra) [reply content/_guides/helpers.md:90]: this is a literal quote though

File: content/_guides/helpers.md [hunk 25]
```diff
@@ -0,0 +1,165 @@
...
+
+If _you_ need the factoid to remind you of the information, and the bot does
+not give factoids in PM, explain as you're doing it. This also teaches them
+how to use the bot. Also, don't just leave them hanging with the answer! Check
+it yourself to ensure it has the information you think it does, and prompt the
+user to confirm if it covers what they want.
+
+## When to stop engaging
+
+When a situation requires intervention and your own attempts to diffuse it do
```
Comment (spb) [inline content/_guides/helpers.md:148]:
```suggestion
When a situation requires intervention and your own attempts to defuse it do
```

File: content/_guides/catalyst.md (hunk 5 above)
Comment (Mikaela) [reply content/_guides/catalyst.md]: Yes, thank you.
